src/main/java/frc/robot/RobotContainer.java

```diff
@@ -208,20 +208,21 @@ public RobotContainer() {
         lastReef = hexSideChooser.getSelected();
         lastBackwards = false;
         rebindAutoPlace = () -> {
+            SuperstructurePreset l = lChooser.getSelected();
             driverController.leftBumper().whileTrue(new AutoPlace(
                 drivetrain,
                 superstructure,
                 scrubber,
                 new Node(
-                    lChooser.getSelected(),
+                    l,
                     getHex(),
                     leftRightChooser.getSelected(),
                     (scrubChooser.getSelected() ? AutoPlace.getScrubPose(getHex()) : SuperstructurePreset.MANUAL_OVERRIDE)
                 ), "", false, false, getBackwards())
                     .finallyDo(() -> {
                         // CommandScheduler.getInstance().schedule(
                         superstructure.setDefaultCommand(
-                            superstructure.setPreset(lChooser.getSelected() == SuperstructurePreset.L1
```

Comment:
This would query the chooser value at the end of the auto and not use the one we started with.